### ISSUE TITLE
stop all

### ISSUE THREAD

**Issue by mmiat** [duplicati/duplicati#3179]
feature request: in the prograss bar, when i click on "x", it stops the current job
it could be useful another link to stop ALL jobs, so i dont need to stop them manually and wait for each

**Comment by Pectojin** [comment]
Since only one job can run at a time "stopping all" would mean clearing the queue for jobs that are scheduled to run *right now*. 

I presume the "expected" behavior is then to have any jobs queued in the future (even 1 minute) still run as expected?
And then any jobs "stopped" are requeued for their next run, correct?

**Comment by mmiat** [comment]
yes, exactly, if i choose "stop all" no more jobs has to be started
next run will be executed as expected, no new behaviour is requested
thanks
